astropy/cosmology/flrw/scalar_inv_efuncs.pyx

```diff
@@ -206,14 +206,14 @@ def fwpwacdm_inv_efunc(double z, double Om0, double Ode0,
 def w0wzcdm_inv_efunc_norel(double z, double Om0, double Ode0, double Ok0,
     double w0, double wz):
   cdef double opz = 1.0 + z
-  cdef Odescl = opz**(3. * (1. + w0 - wz)) * exp(-3. * wz * z)
+  cdef Odescl = opz**(3. * (1. + w0 - wz)) * exp(3. * wz * z)
```

Comment:
`cdef` without a type seems weird, not sure what it does actually, maybe it uses `object` by default ? But anyway it seems this should be `cdef double` instead, same below.